src/main/java/com/sem/btrouble/TestState.java

```diff
@@ -0,0 +1,163 @@
+package com.sem.btrouble;
+
+import com.sem.btrouble.event.PlayerEvent;
+import com.sem.btrouble.model.Drawable;
+import com.sem.btrouble.model.Game;
+import com.sem.btrouble.model.Observer;
+import com.sem.btrouble.model.Player;
+import com.sem.btrouble.model.Room;
+import com.sem.btrouble.model.Rope;
+import org.newdawn.slick.Color;
+import org.newdawn.slick.GameContainer;
+import org.newdawn.slick.Graphics;
+import org.newdawn.slick.Image;
+import org.newdawn.slick.Input;
+import org.newdawn.slick.SlickException;
+import org.newdawn.slick.TrueTypeFont;
+import org.newdawn.slick.geom.Circle;
+import org.newdawn.slick.state.BasicGameState;
+import org.newdawn.slick.state.StateBasedGame;
+import org.newdawn.slick.util.ResourceLoader;
+
+import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Test state
+ */
+public class TestState extends BasicGameState implements Observer {
+    private TrueTypeFont font;
+    private List<Drawable> drawables;
+    private Image background;
+    private Game game;
+    private Player player;
+
+    /**
+     * Initialize method of the slick2d library.
+     *
+     * @param gc
+     *            should be the GameContainer containing the game.
+     * @param sbg
+     *            the reference to the StateBasedGame.
+     * @throws SlickException
+     *             when the game could not be initialized.
+     */
+    @Override
+    public void init(GameContainer gc, StateBasedGame sbg) throws SlickException {
+        // load font from a .ttf file
+        try {
```

Comment:
Deze code zou in een aparte methode kunnen, nu heeft de init namelijk de extra responsibility van het inladen van een font wat beter naar een andere methode kan worden gedelegeerd.